automation_tools/__init__.py

```diff
@@ -425,9 +425,24 @@ def setup_scap_client():
     run('yum -y install puppet-foreman_scap_client', warn_only=True)
 
 
-def install_discovery_image():
-    """Task to install foreman discovery image."""
+def setup_foreman_discovery():
+    """Task to setup foreman discovery."""
+
+    admin_password = os.environ.get('ADMIN_PASSWORD')
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+    template_url = os.environ.get('PXE_DEFAULT_TEMPLATE_URL')
```

Comment:
What if this environment variable is not set? Is `None` a legal value? If so, then this is OK. But if this environment variable is required and no sane default can be given, then just perform a regular dict lookup:

```python
template_url = os.environ['PXE_DEFAULT_TEMPLATE_URL']
```